### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by Rudxain**
Less redundancy, easier to maintain and minify

### CODE REVIEW

**Comment by sinisimattia** [comment]
Dude, please

**Comment by sinisimattia** [comment]
I need this, otherwise i can't complete my project, please merge

**Comment by Rudxain** [comment]
@sinisimattia  What do you mean?

**Comment by ghost** [comment]
Genuis.

File: index.js [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 'use strict';
 
-const lower = 'abcdefghijklmnopqrstuvwxyz';
-const upper = lower.toUpperCase();
+var lower = 'abcdefghijklmnopqrstuvwxyz';
```
Comment (Natizyskunk) [inline index.js]: @Rudxain,
Why changing `const` to `var` ?

Comment (Rudxain) [reply index.js]: @Natizyskunk
> Why changing `const` to `var` ?

To avoid a semver "breaking change". This PR should cause a "patch" increment, but `const` is only compatible with non-legacy Node versions, therefore it requires a "major" increment.

TBF, most repos don't consider new language features to be "breaking", because the public API stays identical.

But even if `version` (in the package.json) isn't major-incremented, the following must happen (if we introduce `const`):
```diff
 "engines": {
-   "node": ">=0.10.0"
+   "node": ">=6.0.0"
 }
```

TLDR: retro-compatibility

**Comment by ghost** [comment]
Why is this PR a thing.

File: index.js (hunk 1 above)
Comment (Natizyskunk) [reply index.js]: @Rudxain,
I see, thanks for your quick reply 🙂

**Comment by Rudxain** [comment]
closing as stale
